docs/security.html

```diff
@@ -116,36 +116,36 @@ <h3 class="anchor-heading"><a id="listener_configuration" class="anchor-link"></
       (i.e. it is just a broker), it must still define the controller listener along with
       any security properties that are needed to configure it. For example, we might
       use the following configuration on a standalone broker:</p>
-      
+
     <pre><code class="language-text">process.roles=broker
 listeners=BROKER://localhost:9092
 inter.broker.listener.name=BROKER
-controller.quorum.voters=0@localhost:9093
+controller.quorum.bootstrap.servers=localhost:9093
 controller.listener.names=CONTROLLER
 listener.security.protocol.map=BROKER:SASL_SSL,CONTROLLER:SASL_SSL</code></pre>
 
     <p>The controller listener is still configured in this example to use the <code>SASL_SSL</code>
       security protocol, but it is not included in <code>listeners</code> since the broker
       does not expose the controller listener itself. The port that will be used in this case
-      comes from the <code>controller.quorum.voters</code> configuration, which defines
+      comes from the <code>controller.quorum.bootstrap.servers</code> configuration, which defines
       the complete list of controllers.</p>
 
     <p>For KRaft servers which have both the broker and controller role enabled, the configuration
       is similar. The only difference is that the controller listener must be included in
       <code>listeners</code>:</p>
-    
+
     <pre><code class="language-text">process.roles=broker,controller
 listeners=BROKER://localhost:9092,CONTROLLER://localhost:9093
 inter.broker.listener.name=BROKER
-controller.quorum.voters=0@localhost:9093
+controller.quorum.bootstrap.servers=localhost:9093
 controller.listener.names=CONTROLLER
 listener.security.protocol.map=BROKER:SASL_SSL,CONTROLLER:SASL_SSL</code></pre>
 
-    <p>It is a requirement for the port defined in <code>controller.quorum.voters</code> to
-      exactly match one of the exposed controller listeners. For example, here the
-      <code>CONTROLLER</code> listener is bound to port 9093. The connection string
-      defined by <code>controller.quorum.voters</code> must then also use port 9093,
-      as it does here.</p>
+    <p>It is a requirement for the host and port defined in <code>controller.quorum.bootstrap.servers</code>
+      is routed to the exposed controller listeners. For example, here the <code>CONTROLLER</code>
```

Comment:
`It is a requirement that the... is routed to`

Reply:
Fixed